config/default/manager/manager.yaml

```diff
@@ -189,3 +199,13 @@ webhooks:
         apiGroups: ["kubevirt.io"]
         apiVersions: ["v1alpha3"]
         resources: ["virtualmachines"]
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: service
```

Comment:
is this the secret added in the kube-addmissions? If so, then maybe the name is too general? what do you think?

Reply:
forgot about kustomize adding the prefix. closing this..